### ISSUE TITLE
I try to train codeformer stage I , but D loss always to 0

### ISSUE THREAD

**Issue by kelisiya** [sczhou/CodeFormer#77]
I use V100 GPU and batch is 1  , the loss and network same as paper , and my code reference GPEN training code .
